services/frontend/src/components/AvatarIcon/AvatarIcon.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {Avatar, Tooltip} from "@material-ui/core";
+import "./AvatarIcon.scss";
+
+const AvatarIcon = ({name, role}) => (
```

Comment:
Now that we have this nice reusable component, it should replace the Avatar on the `Profile` page. Especially considering the logic was taken from there...